Makefile

```diff
@@ -96,20 +97,23 @@ verify.tidy:
 	./hack/verify-tidy.sh
 
 .PHONY: verify.manifests
-verify.manifests:
+verify.manifests: kustomize
 	./hack/verify-manifests.sh
 
 # ------------------------------------------------------------------------------
 # Build - Manifests
 # ------------------------------------------------------------------------------
 
 .PHONY: manifests
-manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
-	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
-	go run hack/generators/manifests/main.go --directory config/crd/bases/
+manifests: manifests.crds manifests.single
+
+.PHONY: manifests.crds
+manifests.crds: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
+	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=build/config/crd/bases
+	go run hack/generators/manifests/main.go --input-directory build/config/crd/bases/ --output-directory config/crd/bases
 
 .PHONY: manifests.single
-manifests.single: ## Compose single-file deployment manifests from building blocks
+manifests.single: kustomize ## Compose single-file deployment manifests from building blocks
```

Comment:
```suggestion
manifests.single-v2: kustomize ## Compose single-file deployment manifests from building blocks
```

Reply:
ditto - see my suggested approach. This can hopefully help keep the scope of this PR under control, and also account for the proposal @ccfishk is making.